ansible/roles/ocp-workload-optaweb-vrp/tasks/workload.yml

```diff
@@ -1,51 +1,54 @@
 ---
 - name: define ocp_project
   set_fact:
-    ocp_project: "optaweb-vrp-{{guid}}"
+    ocp_project: "optaweb-vrp-{{ guid }}"
 
     # Templates come from here: https://raw.githubusercontent.com/jorgemoralespou/ose-sample-apps-layouts
 
 - name: Create project for OptaPlanner Vehicle Routing Demo
   shell: |
-         oc new-project {{ocp_project}} \
+         oc new-project {{ ocp_project }} \
          --display-name="OptaPlanner Vehicle Routing Demo" \
          --description="OptaPlanner Vehicle Routing Demo"
   ignore_errors: true
 
 - name: "Label namespace"
-  command: "oc label namespace {{ocp_project}} AAD='{{guid}}'"
+  command: "oc label namespace {{ ocp_project }} AAD='{{ guid }}'"
 
 #- name: Set project limit LimitRange
-#  shell: "oc create -f /tmp/{{guid}}//limit-range.yaml -n {{ocp_project}}"
+#  shell: "oc create -f /tmp/{{ guid }}//limit-range.yaml -n {{ ocp_project }}"
 
 # Check whether the required ImageStreams are available in the "openshift" namespace.
 # If they are not, we import them.
 # We need to do this in the "openshift" namespace, in order to have the secret to pull from registry.redhat.io.
 
 - name: "Import ReactJS WebApp Template"
-  shell: "oc create -f https://raw.githubusercontent.com/DuncanDoyle/reactjs-openshift/master/create-react-app-openshift-template.yaml -n {{ocp_project}}"
+  shell: >-
+    oc create -f
+    https://raw.githubusercontent.com/DuncanDoyle/reactjs-openshift/master/create-react-app-openshift-template.yaml
+    -n {{ ocp_project }}
 
 - name: "Create OptaWeb backend app"
-  shell: "oc new-app quay.io/optaplanner/optaweb-vrp-backend -n {{ocp_project}}"
+  shell: "oc new-app quay.io/optaplanner/optaweb-vrp-backend -n {{ ocp_project }}"
 
 - name: "Expose backend service"
-  shell: "oc expose svc/optaweb-vrp-backend -n {{ocp_project}}"
+  shell: "oc expose svc/optaweb-vrp-backend -n {{ ocp_project }}"
 
 - name: "Get OptaWeb backend Route"
-  shell: "oc get route optaweb-vrp-backend -n {{ocp_project}} | awk 'FNR > 1 {print $2}'"
+  shell: "oc get route optaweb-vrp-backend -n {{ ocp_project }} | awk 'FNR > 1 {print $2}'"
   register: oc_get_route_output
 
 - name: "Set OptaWeb backend Route fact"
   set_fact:
     optaweb_vrp_backend_route: "{{ oc_get_route_output.stdout }}"
 
 - name: "Create OptaWeb VRP front-end ReactJS app"
-  shell: "oc new-app --name=optaweb-vrp-frontend -e REACT_APP_BACKEND_URL=http://{{optaweb_vrp_backend_route}} \
+  shell: "oc new-app --name=optaweb-vrp-frontend -e REACT_APP_BACKEND_URL=http://{{ optaweb_vrp_backend_route }} \
```

Comment:
this should be corrected to maintain consistency with other uses of multiline `shell` in this file